data/VISHAL3545.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Vishal Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Data Engineer turned *Content Creator*",
+  "avatar": "https://github.com/VISHAL3545.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vishal304094",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect me on LinkedIn",
+      "url": "https://www.linkedin.com/in/vishal304094/",
+      "icon": "FaLinkedIn"
```

There is an invalid icon, it should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
```

---

@kumarsonsoff3 good spot! Given the length of time since your comment and as this is not a blocker,  I will be fixing the avatar point (see below) and then merging. 

@VISHAL3545 if you wish to update this then I suggest you raise a new PR to edit your Profile.  If you do so, you can take a look at the editing guide in docs: https://linkfree.eddiehub.io/docs/how-to-guides/editing and also the available icons: https://linkfree.eddiehub.io/icons